presto-hive/src/test/java/com/facebook/presto/hive/TestHiveDistributedQueriesWithOptimizedRepartitioning.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive;
+
+import com.facebook.presto.testing.MaterializedResult;
+import com.facebook.presto.tests.AbstractTestDistributedQueries;
+import com.google.common.collect.ImmutableMap;
+import org.testng.annotations.Test;
+
+import java.util.Optional;
+
+import static com.facebook.presto.sql.tree.ExplainType.Type.LOGICAL;
+import static com.google.common.collect.Iterables.getOnlyElement;
+import static io.airlift.tpch.TpchTable.getTables;
+import static org.testng.Assert.assertEquals;
+
+public class TestHiveDistributedQueriesWithOptimizedRepartitioning
+        extends AbstractTestDistributedQueries
+{
+    public TestHiveDistributedQueriesWithOptimizedRepartitioning()
+    {
+        super(() -> HiveQueryRunner.createQueryRunner(
+                getTables(),
+                ImmutableMap.of("experimental.optimized-repartitioning", "true",
+                        "driver.max-page-partitioning-buffer-size", "200B"),
+                Optional.empty()));
+    }
+
+    @Override
+    protected boolean supportsNotNullColumns()
+    {
+        return false;
+    }
+
+    @Override
+    public void testDelete()
+    {
+        // Hive connector currently does not support row-by-row delete
+    }
+
+    @Test
+    public void testExplainOfCreateTableAs()
```

Comment:
What's this test?

Reply:
ah I lost track of why it is here. Seems it doesn't belong here. I just deleted it.